dash/dash.py

```diff
@@ -320,6 +321,7 @@ def serve_layout(self):
 
     def _config(self):
         config = {
+            'multi_output': True,
```

Comment:
Is the idea to make this configurable by the user?

Reply:
Or is this for backward compatibility?

Reply:
It's for backward compatibility.